spec/models/miq_ae_class_copy_spec.rb

```diff
@@ -105,7 +105,7 @@
       new_ids = [miq_ae_class.id] * ids.length
       expect(miq_ae_class_copy).to receive(:to_domain).with(domain, nil, false).exactly(ids.length).times { miq_ae_class }
       expect(miq_ae_class).to receive(:fqname).with(no_args).exactly(ids.length).times { fqname }
-      expect(MiqAeClass).to receive(:find).with(an_instance_of(Fixnum)).exactly(ids.length).times { miq_ae_class }
+      expect(MiqAeClass).to receive(:find).with(an_instance_of(Integer)).exactly(ids.length).times { miq_ae_class }
```

Comment:
@djberg96 This is failing tests on Ruby 2.3 since `1` is not an `instance_of` but only a `kind_of?` on that version.

```
irb(main):012:0> RUBY_VERSION
=> "2.3.8"

irb(main):013:0> 1.instance_of?(Fixnum)
=> true
irb(main):014:0> 1.instance_of?(Integer)
=> false

irb(main):015:0> 1.kind_of?(Fixnum)
=> true
irb(main):016:0> 1.kind_of?(Integer)
=> true
```

Therefore you get the clear error message:
```
expected: (an instance of Integer)
     got: (1)
```
🙁